cmd/context/delete.go

```diff
@@ -32,8 +32,12 @@ func DeleteCMD() *cobra.Command {
 		Short: "Delete a context",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := context.Background()
-			args[0] = okteto.AddSchema(args[0])
-			args[0] = strings.TrimSuffix(args[0], "/")
+			// only format the context name if sure it is a URL
+			if strings.Count(args[0], ".") == 2 {
```

Comment:
We can't be sure that this expression means that it's an URL. Could we use [isUrl](https://github.com/okteto/okteto/blob/007a3a47c10b00ee4dfaed1c052427c213449428/cmd/context/create.go#L353) or should it be tweaked to make it work?

Reply:
`isUrl` seems to be returning `false` for `cloud.okteto.com` as an argument. 
<img width="883" alt="image" src="https://user-images.githubusercontent.com/56963264/188222982-a21b0c55-7ede-43ac-a4f5-6b2bc77b751a.png">

Reply:
> We can't be sure that this expression means that it's an URL.

What other way can we use to check?

Reply:
Sorry, I didn't load the context and told you something that was wrong 😢. We can try to check if `args[0]` [is on the context list as it is](https://github.com/okteto/okteto/blob/master/pkg/okteto/context.go#L44) If the context doesn't exist we should transform it to an url to check if it was an okteto cluster